tests/fileTypeTests.py

```diff
@@ -119,7 +119,7 @@ def test_approp_mixed(self):
         """Test mixed job with some rows failing."""
         jobId = self.jobIdDict["mixed"]
         self.passed = self.run_test(
-            jobId, 200, "finished", 6330, 4, "complete", 46, True)
+            jobId, 200, "finished", 8847, 4, "complete", 46, True)
```

Comment:
In moving to sqlRules.csv, I put in the complete description from the prac & proc document, my old version had a much simpler message about not finding TAS in the CARS file.  The much longer error message led to a larger file size.